docs/topics/whatsnew13.md

```diff
@@ -403,7 +403,7 @@ efficient and easy to explore alternative: `keys.associateWith { getValue(it) }`
 fun main() {
 //sampleStart
     val keys = 'a'..'f'
-    val map = keys.associateWith { it.toString().repeat(5).capitalize() }
+    val map = keys.associateWith { it.toString().repeat(5).replaceFirstChar { it.uppercase() } }
```

Comment:
This may not work with `kotlin-min-compiler-version="1.3"` and also not appropriate as an example of what's new in 1.3, because there were no such functions `uppercase` and `replaceFirstChar` in 1.3. Better to find another transform.

Reply:
I've decided just to make this block not runnable — this doc is about 1.3, so let leave it with the code that works for that version.

Reply:
ok, but `capitalize` is not essential for this example, can be replaced with something else.